src/main/resources/view/AccountCard.fxml

```diff
@@ -30,11 +30,11 @@
             <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
             <Label fx:id="description" styleClass="cell_small_label" text="\$description" />
         </VBox>
-        <VBox alignment="CENTER" minHeight="105" GridPane.columnIndex="2">
+        <VBox alignment="CENTER_LEFT" minHeight="105" GridPane.columnIndex="2">
             <padding>
                 <Insets top="5" right="5" bottom="5" left="15" />
             </padding>
-            <Label fx:id="status" styleClass="cell_big_label"/>
+            <Label fx:id="balance" text="Your balance is $\$first" styleClass="cell_big_label" />
```

Comment:
By showing the balance in a column to to the right, it's possible that if the user minimizes the window enough, they will be unable to see the balance (this is what happened for loans' status icons). I suggest moving this into the original account column as a label. Maybe in between the account's name and description?

Reply:
Thanks for the advice. I have already shifted the position of the balance.